src/main/java/org/apache/sysml/runtime/controlprogram/context/SparkExecutionContext.java

```diff
@@ -260,12 +263,36 @@ public static SparkConf createSystemMLSparkConf() {
 		
 		return conf;
 	}
-	
+
+	/**
+	 * Create a SystemML-preferred Spark Session.
+	 * 
+	 * @param appName the application name
+	 * @param master the master value (ie, "local", etc)
+	 * @return
+	 */
+	public static SparkSession createSystemMLSparkSession(String appName, String master) {
```

Comment:
Could we move this function into the `test` package?  Looks great for encapsulating a good default spark session for our test suites, but I'm afraid it could cause confusion later if it's left inside the runtime codebase.